build.gradle

```diff
@@ -176,7 +176,7 @@ if (rootProject.hasProperty('buildPlatforms')) {
 } else {
     if (!rootProject.hasProperty('nativeCompile')) {
         rootProject.ext.platforms = [
-                'linux-386', 'linux-amd64',
+                'linux-386', 'linux-amd64', 'linux-arm64',
```

Comment:
linux-arm64 is already in the build matrix just one line (180) below where you added it.